core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java

```diff
@@ -214,6 +214,11 @@ public static SiteConfiguration auto() {
 
   private SiteConfiguration(Map<String,String> config) {
     ConfigCheckUtil.validate(config.entrySet(), "site config");
+    config.forEach((prop, value) -> {
+      if (prop.startsWith(Property.TABLE_PREFIX.getKey())) {
+        log.warn("Setting table props in site configuration is deprecated, saw {}", prop);
```

Comment:
We call it SiteConfiguration internally, but it no longer has the word "site" in it (at least not by default) in any of the user-facing stuff. The log message could display `accumulo.properties` instead, or if it knows the actual filename used to construct this SiteConfiguration, it could display that, or it could say something like "configuration files" or "startup config files". Newcomers may not understand what a "site file" is, because they didn't work with Accumulo when the file was called `accumulo-site.xml`

Also, I'm okay with being noisy here, but if you wanted to be less noisy, you could use a stream collector to construct a message about all the violations, and just emit one warning instead of many.

Reply:
The shell config still uses the `site`  naming.
```
root@uno> config -f table.durability
-----------+---------------------+----------------------------------------------------------------------------------------------------------------
SCOPE      | NAME                | VALUE
-----------+---------------------+----------------------------------------------------------------------------------------------------------------
default    | table.durability .. | sync
site       |    @override ...... | flush
system     |    @override ...... | sync
-----------+---------------------+----------------------------------------------------------------------------------------------------------------
```